### ISSUE TITLE
about tse

### ISSUE THREAD

**Issue by zuowanbushiwo** [mcw519/PureSound#1]
你好
      请问 [skim_causal_460_wNoise_IS_tsdr.ckpt](https://github.com/mcw519/PureSound/blob/main/egs/tse/demo/skim_causal_460_wNoise_IS_tsdr.ckpt) 和 [libri2mix_max_2spk_clean_16k_1c.ckpt](https://github.com/mcw519/PureSound/blob/main/egs/tse/pretrained/tse/libri2mix_max_2spk_clean_16k_1c.ckpt) 这两个模型有什么不一样吗？ 我用Netron 查看这两个模型，发现它们参数的结构都一样，只有loss 函数有一些不一样。skim_causal_460_wNoise_IS_tsdr.ckpt 效果是要好一些吗？使用了更多的数据训练？
谢谢！

**Comment by mcw519** [comment]
Hi,

skim_causal_460_wNoise_IS_tsdr.ckpt trained by LibriSpeech 100 and 360 clean set and also adding noise.
libri2mix_max_2spk_clean_16k_1c.ckpt trained only by LibriSpeech 100 clean set without adding noise.

And yes, skim_causal_460_wNoise_IS_tsdr.ckpt utilzed tSDR as loss function rather than Si-SDR to make sure the consistence of output signal gain.

thanks.

**Comment by zuowanbushiwo** [comment]
Hi 
Thank you very much for your reply,
is skim_causal_460_wNoise_IS_tsdr.ckpt also only suitable for 2-speaker mix?
Thanks！

**Comment by mcw519** [comment]
Hi,
In general yes, because it trained by 2-mix data.
thanks.

**Comment by zuowanbushiwo** [comment]
Hi 
    Thank you very much for your reply， One more question, consider the case of 2 speakers (one target speaker, one interfering speaker), also with some noise. Is it possible to separate the voice of the target speaker and the voice of the interfering speaker at the same time? This may be an ordinary speech separation problem, but I don't know if the separation will be more accurate when one speaker is known. 
Thanks！

**Comment by zuowanbushiwo** [comment]
Hi 
     Which of the following causal models works best? Thanks

```
td_tse_conv_tasnet_v0_causal
tse_unet_tcn_v0_causal
tse_skim_v0_causal
tse_skim_v1_causal
tse_skim_v2_causal
```

**Comment by mcw519** [comment]
Hi,
There are some records in the [readme](https://github.com/mcw519/PureSound/blob/main/egs/tse/README.md) page. you can check it there.
thanks.

**Comment by zuowanbushiwo** [comment]
ok thanks. I read this readme, but there are only td_tse_conv_tasnet_v0_causal and tse_skim_v0_causal results.

**Comment by mcw519** [comment]
hi,
I think the tse_skim_v0_causal is the one you need.
thanks.
